README.md

```diff
@@ -8,6 +8,25 @@ However, on-device debugging is hard, so the engine can also be compiled to run
 
 ## Building
 
+### Using Docker (Easiest for beginners)
+
+#### Prerequisites:
+ - Working Linux/MacOS system with Docker environment
+ - Dependencies commands:
+   - docker
+   - echo
+   - whoami
+   - grep
+   - chcon (Linux only)
+
+#### Building
+ - Clone the repository
+ - Change code as desired
+ - Change into the spade directory
+ - Run `build-with-docker.sh`
```

Comment:
imo a simple "Build and run the docker image with ./build-with-docker.sh" will do

Reply:
also you will probably change into spade directory before changing code here